File: fizzbuzz.R
fizzbuzz <- function(n) {
  output <- c()
  if (n %% 3 == 0 && n %% 5 == 0) {
    output[n] <- "FizzBuzz"
  } else if (n %% 3 == 0) {
    output[n] <- "Fizz"
  } else if (n %% 5 == 0) {
    output[n] <- "Buzz"
  } else {
    output[n] <- "None"
  }
}

lapply(c(1:15), fizzbuzz)

File: fizzbuzz/fizzbuzz.cpp
#include <iostream>
 
int main() {
 
  // Brain explodes here:
  for (int i=0; i <= 100; i++){

    if (i % 3 == 0 && i % 5 == 0) {
      std::cout << "FizzBuzz \n";
    }  

    if (i % 3 == 0) {
      std::cout << "Fizz \n";
    }  

    if (i % 5 == 0) {
    std::cout << "Buzz \n";
    }  

    else {
      std::cout << i << " None \n";
    }

  }

}
